src/presentation/styles/framerMotionComponents/Modal.js

```diff
@@ -0,0 +1,65 @@
+import React, { useState } from "react";
+import { Link, useNavigate } from 'react-router-dom';
+import { motion, AnimatePresence } from 'framer-motion';
+import * as animationConstants from './AnimationConstants.js'
+import '../styleSheets/css/Modal.css'
+import { HoverButtonClick, HoverButtonNoClick } from './HoverButton.js'
+import { createVFBlock } from "../../../controllers/UserRequests.js";
+import EndPage from "../../pages/EndPage.js";
+
+
+const Modal = ({ showModal, closer, currIntent }) => {
+    const navigate = useNavigate()
+    function redirectToEndPage() {
+        createVFBlock(currIntent)
+        navigate("/endpage")
+    }
+    return (
+        <AnimatePresence>
+            {showModal && (
+                <motion.div className="backdrop"
+                    variants={animationConstants.backdrop}
+                    initial="hidden"
+                    animate="visible">
+                    <motion.div className="modal" variants={animationConstants.modal}>
+                        <button type="button" className="btn-close btn-close-white btn-block" aria-label="Close" onClick={closer}></button>
+                        <h1>Let's Make Your Chatbot Pop!</h1>
+                        <h2>Almost there! Just input your information below and we'll make your prompt for you!</h2>
```

Comment:
I love the "personality" we've choose for guiding the user through the process. By taking a really friendly stance It kind of distracts the users from the fact that were asking for more of their information